src/lib/y2network/sysconfig/connection_config_writers/dummy.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/sysconfig/connection_config_writers/base"
+
+module Y2Network
+  module Sysconfig
+    module ConnectionConfigWriters
+      # This class is responsible for writing the information from a ConnectionConfig::Dummy
+      # object to the underlying system.
+      class Dummy < Base
+      private
+
+        # @see Y2Network::ConnectionConfigWriters::Base#update_file
+        def update_file(_conn)
+          file.interfacetype = "dummy"
```

Comment:
I though that this is discouraged by wicked people

Reply:
no, dummy is the "only" (ok there is one more exception, but ... ;-) type when you have to specify it.

Reply:
Perhaps we could add a comment there to clarify that it is an exception (so we do not remove it in the future during a clean-up :) ).

Reply:
Not for dummy interfaces. It is also part of the man (man ifcfg-dummy), basically in case of dummy interfaces, them can only be inferred by the interface name "dummyX" or by the 'interfacetype' attribute.

Reply:
see https://bugzilla.suse.com/show_bug.cgi?id=1129552